app/README-app.md

```diff
@@ -22,10 +113,35 @@ See [here](https://docs.getmontecarlo.com/docs/platform-architecture) for archit
 ### Configure the agent
 Once the scripts described in the previous steps were executed you can proceed to setup the application itself.
 The only configuration required is the authentication credentials that will be used to connect to Monte Carlo:
-- Execute the Snowflake Application.
-- Enter the Key ID and Secret you got from Monte Carlo and click on "Configure".
-- This will save the credentials in a Snowflake secret and start the Snowpark Container Service used by the application.
-- Switch to the "Advanced" tab and click on "Container Status", it will take a few minutes for the container to be available.
+- Execute the Snowflake Application:
+  - Grant the required permissions.
+  - Grant access to the external access integration.
+  - Launch the application
+- Click "Update Token":
+  - Enter the Key ID and Secret you got from Monte Carlo and click on "Update Token".
+  - This will save the credentials in a Snowflake secret and start the Snowpark Container Service used by the application.
+- Click on "Container Status", it will take a few minutes for the container to be available.
 - Once the container status is "Ready", the agent is ready to be used.
 - You can use the "Reachability Test" button to check connectivity with Monte Carlo cloud services using the provided credentials.
 
+## Usage Snippets
+### Restart the Service:
+```sql
+CALL MCD_AGENT.APP_PUBLIC.RESTART_SERVICE();
+```
+
+### Update the token
+```sql
+CALL MCD_AGENT.APP_PUBLIC.UPDATE_TOKEN('KEY_ID', 'KEY_SECRET');
```

Comment:
Shouldn't you note that this requires restarting the service to take effect? And that you can do it from the UI.